gapis/service/service.proto

```diff
@@ -1481,7 +1481,14 @@ message ValidateDeviceRequest {
 }
 
 message ValidateDeviceResponse {
+  // Error in validating device.
   Error error = 1;
+
+  // Error from downloading trace file.
+  Error download_error = 2;
```

Comment:
What does "download" mean in this context and why is it separate from the regular trace error?

Reply:
Wanted to separate actual validation error from one that came from temp file creation or copying over the perfetto trace. It felt weird that failing to create temp file or copy perfetto data would cause validation to fail.